phenopacketlab-core/src/main/java/org/monarchinitiative/phenopacketlab/core/ConceptConstantsServiceConfigurer.java

```diff
@@ -83,6 +85,13 @@ private static List<IdentifiedConcept> configureSexConstants(ConceptResourceServ
         return concepts;
     }
 
+    /**
+     * @deprecated
+     * Use {@link #configureGenderConstants()} instead which returns LOINC terms.
+     * @param resourceService resource
+     * @return GSSO terms
+     */
+    @Deprecated
```

Comment:
Private methods are not subject to deprecation since they do not change the public API.